README.md

```diff
@@ -4,7 +4,15 @@ This is the parser for the [RecordFlux](https://github.com/Componolit/RecordFlux
 
 ## Development
 
-As a prerequisite to build the parser, [GNAT Community Edition](https://www.adacore.com/download) >= 2020 needs to be installed. To run the tests, the RecordFlux language package and its dependencies must also be installed. Either use the respective make target:
+As a prerequisite to build the parser, the following dependencies need to be installed:
+
+- [GNAT Community Edition](https://www.adacore.com/download) >= 2020
+- patchelf
+- libgmp-dev (Debian) or gmp-devel (Fedora)
+
+Add `<GNAT-directory>/bin` to the PATH environment variable. It is recommended to prepend it to the existing $PATH.
```

Comment:
What about making it even more specific:

```
**Note**: GNAT must be added to the PATH environment variable after installation: `export PATH=<GNAT-directory>/bin:$PATH`.
```

I would also like to have the "Note: ", because adapting the PATH is actually part of the GNAT installation and not specific to the parser.

Reply:
Changed that